front/app/components/ErrorToast/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Icon, colors, Text } from '@citizenlab/cl2-component-library';
+import React from 'react';
+import styled from 'styled-components';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+const StyledBox = styled(Box)`
+  transform: translateY(-100%);
+  opacity: 0;
+  transition: all 0.8s ease-in-out;
+
+  &.visible {
+    transform: translateY(0%);
+    opacity: 1;
+  }
+`;
+
+type ErrorToastProps = {
+  errorMessage: string;
+  showError: boolean;
+  onClose: () => void;
+};
+
+const ErrorToast = ({ errorMessage, showError, onClose }: ErrorToastProps) => {
```

Comment:
It's possible we'll need to add another prop for the translateY when we add this to the IdeaPage as well :thinking: